google-cloud-pub-sub/src/test/resources/application.conf

```diff
@@ -29,7 +29,7 @@ pekko.connectors.google {
     provider = none
     none {
       project-id = "pekko-connectors"
-      token = "ya29.Elz4A2XkfGKJ4CoS5x_umUBHsvjGdeWQzu6gRRCnNXI0fuIyoDP_6aYktBQEOI4YAhLNgUl2OpxWQaN8Z3hd5YfFw1y4EGAtr2o28vSID-c8ul_xxHuudE7RmhH9sg"
+      token = "TESTTOKEN"
```

Comment:
`TEST_TOKEN` ?

Reply:
@mdedetrich suggested this `TESTTOKEN` value because we already have `TESTSECRET` and some similar values.

This is just test code but the aim is to head off any risk that bounty hunters will find a real looking token and start reporting it.